lib/github-pages-health-check/domain.rb

```diff
@@ -245,13 +245,17 @@ def cname_to_github_user_domain?
 
       # Check if the CNAME points to a Domain that points to pages
       # e.g. CNAME -> Domain -> Pages
+      # rubocop:disable Metrics/AbcSize
       def cname_to_domain_to_pages?
+        return false unless dns?
```

Comment:
@tsusdere This should resolve the issue I am getting 🤞 

<img width="996" alt="image" src="https://github.com/user-attachments/assets/02208b87-30f4-4946-84d4-b72539d7de03">